source/application/REIXS/REIXSAppController.cpp

```diff
@@ -177,11 +160,6 @@ void REIXSAppController::setupScanConfigurations()
 	genericScanConfiguration_ = new AMGenericStepScanConfiguration();
 }
 
-void REIXSAppController::setupUserConfiguration()
```

Comment:
yes, it is generalized in the CLSAppController. This function has the same implementation in all the AppControllers